docs/modules/ROOT/pages/explanation/decisions/access-metering-data.adoc

```diff
@@ -0,0 +1,42 @@
+= Access metering data to run reports
+
+== Problem
+
+As outlined in xref:appuio-cloud:ROOT:references/architecture/metering.adoc[Metering of resource usage], metering data is collected into an instance of Thanos (Prometheus API).
+That data is supposed to be used to generate invoices.
+It might also be used to show cost reports on the {portal}.
+
+The Prometheus data is missing information.
+For example prices and discounts.
+This data has to be augmented using another source.
+
+== Proposals
+
+. Directly use the Prometheus API
++
+The Prometheus API allows to get results for a single point in time.
+It also allows to query all results within a given time range with a data point as defined by a given interval.
+
+. Use of an intermediate storage
++
+The Prometheus data is queried regularly and stored into an intermediate storage.
+That intermediate storage could then also contain data about pricing and discounts.
```

Comment:
I would mention in the problem section that we are solving a data warehousing problem where we have several "dimensions"
Also, there's no mentioning of any aggregation here. Right now it gives the impression that it just caches the prometheus raw results.
```suggestion
The Prometheus data is queried regularly, aggregated, enriched with product data and stored into an intermediate database.
That intermediate storage could then also contain data about pricing and discounts.
```